src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -390,5 +409,58 @@ private async Task ProcessSearchResultsAsync(IEnumerable<SearchResultEntry> sear
                 await UpdateJobRecordAsync(cancellationToken);
             }
         }
+
+        private async Task<SearchResult> GetGroupPatients(string groupId, List<Tuple<string, string>> queryParametersList, DateTimeOffset groupMembershipTime, CancellationToken cancellationToken)
```

Comment:
nit: Seems like both of these methods can move to the GroupMemberExtractor class?

Reply:
GetGroupPatientIds I agree would be good in the GroupMemberExtractor class. GetGroupPatients I think is too specialized for this export use case and needs the search service which isn't in the extractor class currently.